pkg/proxy/proxy.go

```diff
@@ -85,6 +84,7 @@ func injectOwnerReference(h http.Handler) http.Handler {
 			req.Body = ioutil.NopCloser(bytes.NewBuffer(newBody))
 			req.ContentLength = int64(len(newBody))
 		}
+		req.Header.Del("Authorization")
```

Comment:
Probably good to add a comment here/document it in general. Might throw some people off.